README.md

```diff
@@ -38,11 +38,18 @@ var syncFn = function (a) {
   return 'running: ' + a;
 };
 
-runAsync(syncFn, function (answer) {
+runAsync(syncFn, function (err, answer) {
   console.log(answer); // 'running: sync'
 }, 'sync');
 ```
 
+Changes from 0.x.x
```

Comment:
I'll do the releases notes inside of the release page. No need to have them in the readme.

That being said, we'll need to update the readme to demo/explain the promise feature.